php.ini

```diff
@@ -23,4 +23,9 @@ realpath_cache_ttl = 7200
 ;
 ; Multi store support
 ;
-auto_prepend_file = /app/public/magento-vars.php
\ No newline at end of file
+auto_prepend_file = /app/public/magento-vars.php
+
+:
+; This feature was DEPRECATED in PHP 5.6.0, and REMOVED as of PHP 7.0.0.
+:
+always_populate_raw_post_data = -1
```

Comment:
Still it makes sense to merge it for now, later everything you move from this php.ini to server specific one can be removed.